src/js/Lotto.js

```diff
@@ -0,0 +1,47 @@
+import { WARNING_WHEN_NOT_IN_1000_UNITS, UNIT_PRICE } from "./utils/consts.js";
+
+class Lotto {
+  #price
+  constructor(price) {
+    this.#price = price
+  }
+
+  isPriceValid() {
+    if (this.#price < UNIT_PRICE || this.#price >= (UNIT_PRICE * 1000)) return false
+
+    if ((this.#price % UNIT_PRICE) !== 0) return this.valueInitAndAlertNotUnitOfthousand()
```

Comment:
```suggestion
    if (this.#price % UNIT_PRICE !== 0) {
      this.valueInitAndAlertNotUnitOfthousand();
    }
```
앞선 리뷰를 토대로 이렇게 표현할 수 있도록 작업해보세요!